src/games/strategy/engine/chat/Chat.java

```diff
@@ -269,14 +269,14 @@ public void chatOccured(final String message) {
       if (isIgnored(from)) {
         return;
       }
-      synchronized (m_mutexNodes) {
-        m_chatHistory.add(new ChatMessage(message, from.getName(), false));
-        for (final IChatListener listener : m_listeners) {
+      synchronized (mutexNodes) {
+        chatHistory.add(new ChatMessage(message, from.getName(), false));
+        for (final IChatListener listener : listeners) {
           listener.addMessage(message, from.getName(), false);
         }
         // limit the number of messages in our history.
-        while (m_chatHistory.size() > 1000) {
-          m_chatHistory.remove(0);
+        while (chatHistory.size() > 1000) {
```

Comment:
Magic value! Since its commented I don't see a big need for this to be changed...